zetaclient/chains/base/observer_internal_tracker.go

```diff
@@ -93,11 +96,13 @@ func (ob *Observer) WatchMonitoringError(
 				Str(logs.FieldBallotIndex, monitorErr.Msg.Digest()).
 				Msg("error monitoring inbound vote")
 
+			metrics.TransactionsAddedToInternalTrackerTotal.WithLabelValues(ob.chain.Name).Inc()
 			ob.AddInternalInboundTracker(&monitorErr.Msg)
 		}
 	case <-ctx.Done():
 		logger.Debug().
 			Str(logs.FieldZetaTx, zetaHash).
 			Msg("no monitoring error received, the inbound vote likely succeeded")
+		metrics.SuccessfulInboundVotesTotal.WithLabelValues(ob.chain.Name).Inc()
```

Comment:
this should only trigger once , if the 500k vote internally creates a 7m vote 

We dont know at this point however wether it is the 500k vote that was successful or the 7M one